javascript-typing-tutor/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Typing Tutor</title>
+  <link href="styles.css" type="text/css" rel="stylesheet">
+</head>
+<body>
+  <div class="container">
+    <div class="text">
+      <h1>
+        <span class="highlight">o</span>
+        <span>b</span>
+        <span>j</span>
+        <span>e</span>
+        <span>c</span>
+        <span>t</span>
+        <span>&nbsp;</span>
+        <span>m</span>
+        <span>a</span>
+        <span>n</span>
+        <span>i</span>
+        <span>p</span>
+        <span>u</span>
+        <span>l</span>
+        <span>a</span>
+        <span>t</span>
+        <span>i</span>
+        <span>o</span>
+        <span>n</span>
+        <span>&nbsp;</span>
+        <span>3</span>
+        <span>&nbsp;</span>
+        <span>i</span>
+        <span>s</span>
+        <span>&nbsp;</span>
+        <span>b</span>
+        <span>a</span>
+        <span>d</span>
```

Comment:
What happens if you use regular space characters instead of `&nbsp;`? I imagine it will help simplify your JavaScript if you can user regular space characters